driver/query.feature

```diff
@@ -996,6 +996,327 @@ Feature: Driver Query
     """
 
 
+  @ignore-typedb-http
```

Comment:
Hum do we have any http driver BDD tests right now? If yes, we can do this too right?